### DISCUSSION TITLE
The "Done!" flash now pops up to the middle of the screen. It's a bit intrusive.

### DISCUSSION THREAD

**Discussion by kmcbest** [deckerst/aves#985] [General]
Since a couple of builds ago, the "Done!" flash (after tagging) pops up way higher. It was at the bottom of the screen and now in the middle of it. It not only obstructs the image for a while, but also gets in the way when I want to swipe left and right: I have to swipe it off before actually swipe the image.

Can you revert it back to the old behavior?

**Comment by deckerst** [comment]
Screen recording, and system info (from About > Bug Report) please.

**Reply by kmcbest** [reply]
System information and logs:

Package: deckers.thibault.aves
Installer: com.miui.packageinstaller
Aves version: 1.10.9-play, build 118
Flutter: stable 3.19.5
Android version: 14, API 34
Android build: UP1A.230905.011
Device: Xiaomi 23078RKD5C
Support: dynamic colors=true, geocoder=true, HDR=true
Mobile services: not available
System locales: zh_CN
Storage volumes: /storage/emulated/0/
Storage grants:
/storage/emulated/0/Pictures/, ....
Error reporting: true


https://github.com/deckerst/aves/assets/689023/1df5652e-e143-4254-8319-d96842d7898a
